### ISSUE TITLE
Error decoding packet

### ISSUE THREAD

**Issue by TheoKah** [PaperMC/Travertine#36]
Hi,

We updated our 1.7 bungee proxy to travertine and it worked well except for 1 server (custom modded modpack)

Here is what is printed on the proxy:

> io.netty.handler.codec.DecoderException: Error decoding packet class EntityEffect with contents 
>          +-------------------------------------------------+
>          |  0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f |
> +--------+-------------------------------------------------+----------------+
> |00000000| 1d 00 04 c5 6d 00 00 00 01 00 7f ff             |....m.......    |
> +--------+-------------------------------------------------+----------------+
>         at net.md_5.bungee.protocol.MinecraftDecoder.decode(MinecraftDecoder.java:69)
>         at net.md_5.bungee.protocol.MinecraftDecoder.decode(MinecraftDecoder.java:13)
>         at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:88)
>         at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:373)
>         at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:359)
>         at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:351)
>         at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
>         at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:280)
>         at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:396)
>         at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:248)
>         at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:373)
>         at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:359)
>         at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:351)
>         at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
>         at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:373)
>         at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:359)
>         at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:351)
>         at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1334)
>         at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:373)
>         at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:359)
>         at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:926)
>         at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:1018)
>         at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:402)
>         at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:307)
>         at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:873)
>         at java.lang.Thread.run(Thread.java:745)

I can try and isolate what mod does this, but I'd like to avoid it if the info above are enough

**Comment by Techcable** [comment]
There should be a cause of this exception. Could you please gist the _full_ stack trace?

**Comment by TheoKah** [comment]
It is the full stack trace :/

Here is a selection of everything from the player " InitialHandler has connected" to "UpstreamBridge has disconnected" (IP masked)

I can provide the full log using a private channel if it helps

```
03:57:56 [INFO] [/IP_A:55765] <-> InitialHandler has connected
03:57:56 [INFO] [/IP_B:61708] <-> InitialHandler has connected
03:57:57 [FINE] [WAITINGSERVERCOMPLETE Client -> Bungee][SENDING: HandshakeAck]
03:57:57 [FINE] [COMPLETE Server <- Bungee][RECEIVED: HandshakeAck]
03:57:57 [FINE] [COMPLETE Server -> Bungee][SENDING: HandshakeAck]
03:57:57 [FINE] [PENDINGCOMPLETE Client <- Bungee][RECEIVED: HandshakeAck]
03:57:57 [INFO] [Mr_Wafer|/IP_B:61708] <-> ServerConnector [Lobby] has connected
03:57:57 [FINE] [PENDINGCOMPLETE Client -> Bungee][SENDING: HandshakeAck]
03:57:57 [FINE] [DONE Server <- Bungee][RECEIVED: HandshakeAck]
03:57:57 [INFO] [brickercutiepi|/IP_A:55765] <-> ServerConnector [KitchenSink] has connected
03:57:57 [FINE] [COMPLETE Client -> Bungee][SENDING: HandshakeAck]
03:57:57 [FINE] [DONE Server <- Bungee][RECEIVED: HandshakeAck]
03:57:57 [FINE] [START Server -> Bungee][SENDING: ServerHello]
03:57:57 [FINE] [HELLO Client <- Bungee][RECEIVED: ServerHello]
03:57:57 [FINE] [HELLO Client -> Bungee][SENDING: ClientHello]
03:57:57 [FINE] [HELLO Server <- Bungee][RECEIVED: ClientHello]
03:57:57 [FINE] [HELLO Client -> Bungee][SENDING: ModList]
03:57:57 [FINE] [HELLO Server <- Bungee][RECEIVED: ModList]
03:57:57 [FINE] [HELLO Server -> Bungee][SENDING: ModList]
03:57:57 [FINE] [WAITINGSERVERDATA Client <- Bungee][RECEIVED: ModList]
03:57:57 [FINE] [WAITINGSERVERDATA Client -> Bungee][SENDING: HandshakeAck]
03:57:57 [FINE] [WAITINGCACK Server <- Bungee][RECEIVED: HandshakeAck]
03:57:57 [FINE] [WAITINGCACK Server -> Bungee][SENDING: ModIdData]
03:57:57 [FINE] [WAITINGSERVERCOMPLETE Client <- Bungee][RECEIVED: ModIdData]
03:57:57 [FINE] [WAITINGCACK Server -> Bungee][SENDING: HandshakeAck]
03:57:57 [FINE] [WAITINGSERVERCOMPLETE Client <- Bungee][RECEIVED: HandshakeAck]
03:57:57 [FINE] [WAITINGCACK Server -> Bungee][SENDING: FluidIdMap]
03:57:57 [FINE] [WAITINGSERVERCOMPLETE Client <- Bungee][RECEIVED: FluidIdMap]
03:57:58 [INFO] [/IP_C:49700|Master_Debater] -> UpstreamBridge has disconnected
03:57:58 [FINE] [WAITINGSERVERCOMPLETE Client -> Bungee][SENDING: HandshakeAck]
03:57:58 [FINE] [COMPLETE Server <- Bungee][RECEIVED: HandshakeAck]
03:57:58 [FINE] [COMPLETE Server -> Bungee][SENDING: HandshakeAck]
03:57:58 [FINE] [PENDINGCOMPLETE Client <- Bungee][RECEIVED: HandshakeAck]
03:57:58 [FINE] [PENDINGCOMPLETE Client -> Bungee][SENDING: HandshakeAck]
03:57:58 [FINE] [DONE Server <- Bungee][RECEIVED: HandshakeAck]
03:57:59 [FINE] [COMPLETE Client -> Bungee][SENDING: HandshakeAck]
03:57:59 [FINE] [DONE Server <- Bungee][RECEIVED: HandshakeAck]
03:58:01 [INFO] [/IP_D:54569|SausageJesus] -> UpstreamBridge has disconnected
03:58:05 [SEVERE] [/IP_A:55765|brickercutiepi] <-> DownstreamBridge <-> [KitchenSink] - encountered exception
io.netty.handler.codec.DecoderException: Error decoding packet class EntityEffect with contents          +-------------------------------------------------+
         |  0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f |
+--------+-------------------------------------------------+----------------+
|00000000| 1d 00 04 c5 6d 00 00 00 01 00 7f ff             |....m.......    |
+--------+-------------------------------------------------+----------------+
	at net.md_5.bungee.protocol.MinecraftDecoder.decode(MinecraftDecoder.java:69)
	at net.md_5.bungee.protocol.MinecraftDecoder.decode(MinecraftDecoder.java:13)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:88)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:373)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:359)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:351)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:280)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:396)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:248)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:373)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:359)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:351)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:373)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:359)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:351)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1334)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:373)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:359)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:926)
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:1018)
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:402)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:307)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:873)
	at java.lang.Thread.run(Thread.java:745)
03:58:05 [INFO] [brickercutiepi] disconnected with: §cThe server you were previously on went down, you have been connected to a fallback server
03:58:05 [INFO] [/IP_A:55765|brickercutiepi] <-> DownstreamBridge <-> [KitchenSink] has disconnected
03:58:06 [INFO] [/IP_A:55765|brickercutiepi] -> UpstreamBridge has disconnected
```

**Comment by kamcio96** [comment]
Will you update it? I fixed it https://github.com/WaterfallMC/Travertine/commit/765da9ee2dfe1f0c5f742ba57d0a0ba9f83826f8

**Comment by TheoKah** [comment]
Yep, I'm using the last one ( 1d5e74d )

**Comment by alicheek** [comment]
Still seem to have similar issues @kamcio96 

[00:02:06] [Netty Worker IO Thread #4/INFO]: [/109.149.78.140:56419] <-> InitialHandler has connected
[00:02:06] [Netty Worker IO Thread #4/INFO]: [Insultron|/109.149.78.140:56419] <-> ServerConnector [DW20] has connected
[00:02:06] [Netty Worker IO Thread #4/INFO]: [Insultron|/109.149.78.140:56419] <-> ServerConnector [Lobby] has connected
[00:02:06] [Netty Worker IO Thread #4/INFO]: [Insultron|/109.149.78.140:56419] <-> ServerConnector [DW20] has disconnected
[00:02:29] [Netty Worker IO Thread #4/INFO]: [Insultron|/109.149.78.140:56419] <-> ServerConnector [DW20New] has connected
[00:02:47] [Netty Worker IO Thread #23/INFO]: [/5.69.160.51:52744] <-> InitialHandler has connected
[00:02:48] [Netty Worker IO Thread #23/INFO]: [MrsMahavo|/5.69.160.51:52744] <-> ServerConnector [StoneBlock2] has connected
[00:02:51] [Netty Worker IO Thread #4/INFO]: [Insultron] disconnected with: Timed out
[00:02:51] [Netty Worker IO Thread #4/INFO]: [/109.149.78.140:56419|Insultron] <-> DownstreamBridge <-> [Lobby] has disconnected
[00:02:51] [Netty Worker IO Thread #4/INFO]: [/109.149.78.140:56419|Insultron] -> UpstreamBridge has disconnected
[00:03:00] [Netty Worker IO Thread #14/WARN]: [/68.134.215.192:65195|GoRazorbacks56] -> UpstreamBridge - NativeIoException: syscall:read(..) failed: Connection reset by peer
[00:03:00] [Netty Worker IO Thread #14/INFO]: [GoRazorbacks56] disconnected with: NativeIoException : syscall:read(..) failed: Connection reset by peer @ io.netty.channel.unix.FileDescriptor:-1
[00:03:00] [Netty Worker IO Thread #14/INFO]: [/68.134.215.192:65195|GoRazorbacks56] -> UpstreamBridge has disconnected
[00:03:00] [Netty Worker IO Thread #14/INFO]: [/68.134.215.192:65195|GoRazorbacks56] <-> DownstreamBridge <-> [DW20New] has disconnected
[00:03:00] [Netty Worker IO Thread #4/INFO]: [Insultron|/109.149.78.140:56419] <-> ServerConnector [DW20New] has disconnected
[00:03:18] [Netty Worker IO Thread #17/INFO]: [/173.49.233.207:50947] <-> InitialHandler has connected
[00:03:19] [Netty Worker IO Thread #17/INFO]: [the_original_one|/173.49.233.207:50947] <-> ServerConnector [StoneBlock2] has connected
[00:05:06] [Netty Worker IO Thread #0/INFO]: [/109.149.78.140:56443] <-> InitialHandler has connected
[00:05:06] [Netty Worker IO Thread #0/INFO]: [Insultron|/109.149.78.140:56443] <-> ServerConnector [DW20] has connected
[00:05:06] [Netty Worker IO Thread #0/INFO]: [Insultron|/109.149.78.140:56443] <-> ServerConnector [DW20] has disconnected
[00:05:06] [Netty Worker IO Thread #0/INFO]: [Insultron|/109.149.78.140:56443] <-> ServerConnector [Lobby] has connected
[00:05:19] [Netty Worker IO Thread #12/ERROR]: [/94.239.122.246:59128|Viooo] <-> DownstreamBridge <-> [DW20New] - encountered exception
java.lang.IllegalArgumentException: Unknown meta type 14
	at net.md_5.bungee.entitymap.EntityMap.rewriteMetaVarInt(EntityMap.java:240) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at net.md_5.bungee.entitymap.EntityMap.rewriteMetaVarInt(EntityMap.java:133) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at net.md_5.bungee.entitymap.EntityMap_1_12_1.rewriteClientbound(EntityMap_1_12_1.java:152) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at net.md_5.bungee.entitymap.EntityMap.rewriteClientbound(EntityMap.java:99) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at net.md_5.bungee.connection.DownstreamBridge.handle(DownstreamBridge.java:122) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at net.md_5.bungee.netty.HandlerBoss.channelRead(HandlerBoss.java:111) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:323) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:310) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:426) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:278) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1434) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:965) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:799) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:421) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:321) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:897) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]

**Comment by kamcio96** [comment]
version? Pls format log

**Comment by kamcio96** [comment]
Ohh, @alicheek you are using mods, bungecord and forks don't support metadata for mods

**Comment by alicheek** [comment]
@kamcio96  Travertine version git:Travertine-Bootstrap:1.13-SNAPSHOT:2900c0b:unknown this is the version @TheoKah  merged a fix for 1.7.10 

some reason the logs do not seem to be formatted 

Yes we are using modpacks this more a Waterfall issue?

**Comment by electronicboy** [comment]
More of a bungeecord issue, tracked upstream at https://github.com/PaperMC/Waterfall/issues/136
